lib/rubocop/cop/style/empty_case_condition.rb

```diff
@@ -77,10 +77,14 @@ def correct_multiple_alternative_whens(corrector, when_nodes)
 
             # In `when a; r` we have two children: [a, r].
             # In `when a, b, c; r` we have 4.
-            next unless children.count > 2
+            next unless children.size > 2
```

Comment:
@bbatsov is correct. There is a cop for this. It is enabled by default, but it is incredibly limited in what it will report errors on. We have discussed expanding its scope, but frameworks like `ActiveRecord` don't always follow the same rules as Ruby for `count` and `size` are implemented.